internal/cli/cmd/promote/promote.go

```diff
@@ -122,13 +128,18 @@ func (o *promotionOptions) addFlags(cmd *cobra.Command) {
 			option.StageFlag,
 		),
 	)
+	option.Abort(cmd.Flags(), &o.Abort, false, fmt.Sprintf(
+		"Abort a non-terminal promotion. If set, --%s must be set.", option.NameFlag,
+	))
```

Comment:
`verifyStageOptions` could use this also. Would you mind updating that so that we're more consistent on how the abort flag is defined between these two commands?

Reply:
Ah, doh. That was the whole idea.

Reply:
Now available in b4f3b660cfd4c57eb8fb94fb2ad3db82b4771470.